doc/nrf/releases_and_maturity.rst

```diff
@@ -3,7 +3,17 @@
 Releases and maturity
 #####################
 
-The |NCS| receives regular releases, which introduce new components or fix issues in existing features, according to the criteria for :ref:`versions and revisions <dm-revisions>`.
+The |NCS| receives regular releases, which introduce new components or fix issues in existing features.
+Every release consists of a combination of all included repositories at different revisions.
+
+The versioning scheme adopted is similar to `Semantic versioning`_, but with important semantic differences:
+
+* Every release of the |NCS| is identified with a version string in the ``MAJOR.MINOR.PATCH`` format.
+* Occasionally, ``99`` can be added in place of ``PATCH`` to mark that a given release is happening between two major releases.
```

Comment:
This line is a bit confusing. We always have a .99 version visible in the docs to indicate ongoing development, and that development could turn into a minor or major release. Could we reword this sentence to reflect that better?